drivers/interrupt_controller/Kconfig

```
@@ -45,6 +45,14 @@ config LEON_IRQMP
 	help
 	  GRLIB IRQMP and IRQAMP
 
+if ITE_IT8XXX2_WUC
+
+module = WUC
+module-str = wuc
```

better make it generic for all intc drivers, so maybe:

```suggestion
module = INTC
module-str = intc
```

---

Ok